src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -19,24 +19,75 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The miners building.
+ */
 public class BuildingMiner extends AbstractBuildingWorker
 {
+    /**
+     * The NBT Tag to store the floorBlock
+     */
     private static final    String              TAG_FLOOR_BLOCK         = "floorBlock";
+    /**
+     * The NBT Tag to store the fenceBlock
+     */
     private static final    String              TAG_FENCE_BLOCK         = "fenceBlock";
+    /**
+     * The NBT Tag to store the starting level of the shaft.
+     */
     private static final    String              TAG_STARTING_LEVEL      = "startingLevelShaft";
+    /**
+     * The NBT Tag to store list of levels.
+     */
     private static final    String              TAG_LEVELS              = "levels";
+    /**
+     * The NBT Tag to store if the shaft has been cleared.
+     */
     private static final    String              TAG_CLEARED             = "clearedShaft";
+    /**
+     * The NBT Tag to store the location of the shaft.
+     */
     private static final    String              TAG_SLOCATION           = "shaftLocation";
+    /**
+     * The NBT Tag to store the vector-x of the shaft
+     */
     private static final    String              TAG_VECTORX             = "vectorx";
+    /**
+     * The NBT Tag to store the vector-z of the shaft
+     */
     private static final    String              TAG_VECTORZ             = "vectorz";
+    /**
+     * The NBT Tag to store the location of the cobblestone at the shaft.
+     */
     private static final    String              TAG_CLOCATION           = "cobblelocation";
+    /**
+     * The NBT Tag to store the active node the miner is working on.
+     */
     private static final    String              TAG_ACTIVE              = "activeNodeint";
+    /**
+     * The NBT Tag to store the current level the miner is working in.
+     */
     private static final    String              TAG_CURRENT_LEVEL       = "currentLevel";
+    /**
+     * The NBT Tag to store the starting node.
+     */
     private static final    String              TAG_SN                  = "StartingNode";
+    /**
+     * The NBT Tag to store the location of the ladder.
+     */
     private static final    String              TAG_LLOCATION           = "ladderlocation";
+    /**
+     * The NBT Tag to store if a ladder has been found yet.
+     */
     private static final    String              TAG_LADDER              = "found_ladder";
 
+    /**
+     * Defines the material used for the floor of the shaft.
+     */
     private Block floorBlock = Blocks.planks;
+    /**
+     * Defines the material used for the fence of the shaft.
+     */
     private Block fenceBlock = Blocks.oak_fence;
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)